website/docs/d/vra_region.html.markdown

```diff
@@ -0,0 +1,27 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration"
+description: |-
+  Provides a data lookup for vra_region_enumeration.
+---
+
+# Data Source: vra_region_enumeration
```

Comment:
Page title, description and data source name should be `vra_region` to match with the file name.